.github/workflows/unit.yaml

```diff
@@ -0,0 +1,50 @@
+name: Unit Tests
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pyver: [6, 7, 8, 9]
+    name: Run Unit Tests -- Python 3.${{ matrix.pyver }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.${{ matrix.pyver }}
+      - name: Install tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e py3${{ matrix.pyver }}-unit
+      - name: Upload coverage report artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage-py3${{ matrix.pyver }}
+          path: .coverage.py3${{ matrix.pyver }}-unit
```

Comment:
I think we can skip uploading these artifacts. Coverage will be the same regardless of the python version. We should trust the coverage_report  job for that.

Reply:
Ah ok good point. I think the example that I looked at might have had both Python 2 and 3 support where this kind of combined coverage would need to be leveraged. Since we are assuming that are code will work across each of the four Python 3 versions that we support, as the unit tests stay the same across the versions, I'll simplify this workflow.